rest/src/main/kotlin/request/RequestHandler.kt

```diff
@@ -5,11 +5,16 @@ package dev.kord.rest.request
  */
 interface RequestHandler {
 
+    /**
+     * The Discord bot authorization token used on requests.
+     */
+    val token: String
```

Comment:
I don't really think all request handlers will have concerns over the token, maybe someone would like to handle token related issues else where, is it a requirement for the request handler?

Reply:
Here yes, or we would need to change lots of code, because neither the request builder nor the call method have access to the token, so without changing 500 method calls this was the easiest way

Reply:
No, don't get me wrong, remove the token from the interface and add it to the ktor request handler class. This way, whoever wants to implement their own request handler are not required to use a token-based system

Reply:
RestService doesn't have a KtorRequestHandler http://rice.by.devs-from.asia/u/CWZfCb.png